ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh

```diff
@@ -3,10 +3,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-cat >> "${SHARED_DIR}/install-config.yaml" << EOF
-networking:
-  networkType: OpenShiftSDN
-EOF
+echo "## Install yq"
+curl -L https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
I think you should do 
```
curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Reply:
ok, updated with this. I pinned it because I saw how command line changes happened between [v3 and v4](https://mikefarah.gitbook.io/yq/upgrading-from-v3) such that
some commands no longer worked and wanted to avoid that. but this has `set -e` so if that were to happen this step
should fail and we'd find out right away because the jobs would all die that used this step